ethsigner/signer/multiplatform/src/test/java/tech/pegasys/ethsigner/signer/multiplatform/MultiPlatformSubCommandTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.multiplatform;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.nio.file.Path;
+
+import org.apache.logging.log4j.Level;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import picocli.CommandLine;
+import picocli.CommandLine.ParseResult;
+
+class MultiPlatformSubCommandTest {
+
+  private MultiPlatformSubCommand multiPlatformSubCommand;
+  private CommandLine commandLine;
+
+  @BeforeEach
+  void beforeEach() {
+    multiPlatformSubCommand = new MultiPlatformSubCommand();
+
+    commandLine = new CommandLine(multiPlatformSubCommand);
+    commandLine.setCaseInsensitiveEnumValuesAllowed(true);
+    commandLine.registerConverter(Level.class, Level::valueOf);
+  }
+
+  @Test
+  void parseCommandSuccessfullySetDirectory() {
+    final Path expectedPath = Path.of("/keys/directory/path");
+
+    commandLine.parse("--directory", "/keys/directory/path");
```

Comment:
```suggestion
    commandLine.parse("--directory", expectedPath.toString());
```

Maybe that doesn't _quite_ work - but means you don't have the same string twice.

Reply:
done